packages/blockchain-link/package.json

```diff
@@ -67,6 +67,7 @@
         "worker-loader": "^3.0.8"
     },
     "dependencies": {
+        "@solana/web3.js": "^1.78.4",
```

Comment:
I am not happy about this. Haven't checked yet how this lib is being used but in general we prefer not adding libs. If I see corretly, you only use this  in runtime https://github.com/solana-labs/solana-web3.js/blob/320ccbb/packages/library-legacy/src/publickey.ts#L48 
do we really need to include this lib?

Reply:
> If I see correctly, you only use this in runtime 

I also use `Connection` object which is quite important as it basically implements the whole blockchain API. 
https://github.com/trezor/trezor-suite/blob/334e8ddc4d1e5ca96fe3307f7caf903fb0f2f7a8/packages/blockchain-link/src/workers/solana/index.ts#L99

Reply:
hm, I see 😿 . Lets have it there for now. It should be part of worker bundled file only anyway...